web/src/helpers/redux.js

```diff
@@ -1,3 +1,14 @@
+import {
+  flatMap,
+  keys,
+  sortBy,
+  uniq,
+} from 'lodash';
+
+export function getExchangeKeys(zoneHistory) {
```

Comment:
is this really a redux helper?

Reply:
Good catch! I wasn't very sure where to put it though, created a new helpers file `zones.js` and moved it there in https://github.com/tmrowco/electricitymap-contrib/pull/2171/commits/26885cd6d1f8aff209b3303aedcc919017942b81, hope that's better :)